### PR TITLE
add IdeaInfoOverlay

### PR DESCRIPTION
**Pull request by ZorianaPro**

### CODE REVIEW

File: src/components/IdeaInfoOverlay/IdeaInfoOverlay.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		expect(shallow(<IdeaInfoOverlay />)).toMatchSnapshot();
+	});
+
+	describe('click close button', () => {
+		let overlay;
+
+		beforeEach(() => {
+			overlay = mount(<IdeaInfoOverlay/>);
+			overlay.find('.IdeaInfoOverlay-Close').simulate('click');
+			overlay.update();
```
Comment (yoanad) [inline src/components/IdeaInfoOverlay/IdeaInfoOverlay.test.js:20]: Interesting, why do you want the update? It doesn't do that automatically?

File: src/components/IdeaInfoOverlay/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+					<div className="IdeaInfoOverlay-Container">
+						<div className="IdeaInfoOverlay-Box">
+							<div className="IdeaInfoOverlay-Close" onClick={() => setOpen(false)}>
+								close
+							</div>
+							<div className="IdeaInfoOverlay-Title">
+								Idea Name
+							</div>
+							<div className="IdeaInfoOverlay-Description">
+								Idea description Idea descriptionIdea descriptionIdea descriptionIdea
```
Comment (yoanad) [inline src/components/IdeaInfoOverlay/index.js:23]: Improvement idea: https://baconipsum.com/ 🤣

Comment (ZorianaPro) [reply src/components/IdeaInfoOverlay/index.js:23]: I will fix it with next PR, when I use redux

File: src/components/IdeaInfoOverlay/IdeaInfoOverlay.test.js (hunk 1 above)
Comment (ZorianaPro) [reply src/components/IdeaInfoOverlay/IdeaInfoOverlay.test.js]: u are right. This step is not necessary. Component is updated by itself in this case.

**Pull request merged**